x-pack/plugins/apm/server/lib/search_strategies/correlations/queries/get_prioritized_field_value_pairs.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FIELDS_TO_ADD_AS_CANDIDATE } from '../constants';
+import { hasPrefixToInclude } from '../utils';
+
+import type { FieldValuePairs } from './query_field_value_pairs';
+
+export const getPrioritizedFieldValuePairs = (
+  fieldValuePairs: FieldValuePairs
+) => {
+  const prioritizedFields = [...FIELDS_TO_ADD_AS_CANDIDATE];
+
+  return fieldValuePairs.sort((a, b) => {
+    if (
+      (prioritizedFields.includes(a.field) || hasPrefixToInclude(a.field)) &&
+      !prioritizedFields.includes(b.field) &&
+      !hasPrefixToInclude(b.field)
+    ) {
+      return -1;
+    } else if (
+      !prioritizedFields.includes(a.field) &&
```

Comment:
I think we can improve on the repeated `.includes` checks here. We can assign constants to checks like `prioritizedFields.includes(a.field)` and reuse them again.

Reply:
Tweaked in 0a75e15.